storage/storage.c

```diff
@@ -106,7 +106,7 @@ const uint32_t V0_PIN_EMPTY = 1;
 // up constant storage space.
 #define MAX_WIPE_CODE_LEN 50
 
-#if STORAGE_INSECURE_TESTING_MODE && !PRODUCTION
+#if (STORAGE_INSECURE_TESTING_MODE && !PRODUCTION) || STRETCHED_PIN_COUNT > 1
 #define PIN_ITER_COUNT 1
```

Comment:
TODO: This results in zero iterations, since `10 * (PIN_ITER_COUNT / 10)` iterations are actually performed.

Reply:
It's actually OK. Our PBKDF2 implementation always guarantees one iteration, but it's undocumented, so I agree not very nice.